Automattic/wp-calypso
# Secure your brand

#### Changes proposed in this Pull Request

This change adds the Secure your brand campaign

#### Testing instructions

0. Apply D49664-code
1. The country_code cookie needs to be set to 'US'
2. Keep in mind that the diff above would mean that the Domain Sandbox would operate with .com, .net, .org, and .id TLDs. The Domain Sandbox would be used if the Store Sandbox is enabled. Searching for something.com would properly suggest .com, .org, .co domains, and when the next step is clicked, the .co will be removed from the cart because it's unsupported TLD.

That's why, when searching for a domain, search a .live domain, .UK, .co.uk, .id or just deal with it. Finishing the checkout with the store sandbox would require that the domain searched is not .com, .net, or .org

##### For the `test` group
1. Sign up with a custom domain
2. Make sure the Secure your brand step is visible
<img width="1369" alt="Screenshot 2020-10-14 at 16 09 01" src="https://user-images.githubusercontent.com/82778/95993662-0d5ba780-0e38-11eb-98d0-9d081a7dff39.png">

3. You'll need to sign up twice - once by accepting the offer
<img width="1369" alt="Screenshot 2020-10-14 at 16 07 34" src="https://user-images.githubusercontent.com/82778/95993678-12b8f200-0e38-11eb-8c14-a08506bd44df.png">

4. And once by skipping it. In case the offer is skipped, the cart should only have a domain + plan
5. Try the free domain/plan - there should be no secure your brand upsell.
6. Edge case - it's possible that the domain searched has no suggested domains available. The only way to actually force that error would be to hardcode an error response in the endpoint. It should skip the step once the endpoint returns error.

##### For the `control` group
1. Make sure that signing up with a domain+plan does not render a new "Secure your brand" step

Fixes #
